text-show-instances.cabal

```diff
@@ -143,6 +146,7 @@ library
                      , semigroups            >= 0.16.2 && < 1
                      , tagged                >= 0.4.4  && < 1
                      , text                  >= 0.11.1 && < 1.3
+                     , text-short            >= 0.1    && < 0.1.3
```

Comment:
Use `0.2` as the upper version bounds for `text-short` in order to adhere to the PVP.